config/functions/build-gpu-deb

```diff
@@ -82,12 +82,6 @@ build_gpu_deb() {
 			cp -arf archives/hwpacks/mali/${VENDOR}/fbdev_examples/$LINUX/opengles_20 $pkgdir/usr/share/arm/
 		fi
 
-		mkdir -p $pkgdir/usr/lib/udev/rules.d
```

Comment:
You can't remove this, because for legacy kernel if you remove this, then the mali rule will not be applied.